subsys/net/lib/lwm2m/lwm2m_engine.h

```diff
@@ -9,6 +9,22 @@
 
 #include "lwm2m_object.h"
 
+#define LWM2M_PROTOCOL_VERSION "1.0"
+
+/* LWM2M / CoAP Content-Formats */
+#define LWM2M_FORMAT_PLAIN_TEXT		0
+#define LWM2M_FORMAT_APP_LINK_FORMAT	40
+#define LWM2M_FORMAT_APP_OCTET_STREAM	42
+#define LWM2M_FORMAT_APP_EXI		47
+#define LWM2M_FORMAT_APP_JSON		50
+#define LWM2M_FORMAT_OMA_PLAIN_TEXT	1541
```

Comment:
If using LWM2M_FORMAT_OMA_PLAIN_TEXT is undesirable.  Should we remove the define entirely?